src/Components/OurTeam/MainHeading.jsx

```diff
@@ -0,0 +1,10 @@
+import './MainHeading.css'
+
+const MainHeading = ({Heading}) => {
```

Comment:
Change name from MainHeading. Note that all css files will come together and later cause issues if you use same name like "Heading"
make sure it is unique. 
Refer CSS from home page for naming conventions